Lib/test/test_compile.py

```diff
@@ -1231,7 +1232,9 @@ def f():
             with self.subTest(body):
                 namespace = {}
                 source = textwrap.dedent(source_template.format(body))
-                exec(source, namespace)
+                with warnings.catch_warnings():
+                    warnings.simplefilter('ignore', SyntaxWarning)
```

Comment:
Is there value in asserting that the warning is emitted rather than just ignoring it?  (Or is that already tested by another test elsewhere?)

Reply:
It's covered in test_grammar: 

https://github.com/python/cpython/blob/caa2a9799a47294441e4206037620322eea9ed06/Lib/test/test_grammar.py#L1476